modules/ossm-rn-known-issues.adoc

```diff
@@ -35,9 +35,7 @@ These are the known issues in {SMProductName}:
 * link:https://github.com/istio/istio/issues/14743[Istio-14743] Due to limitations in the version of Istio that this release of {SMProductName} is based on, there may be applications that are currently incompatible with {SMProductShortName}. See the linked community issue for details.
 
 * https://issues.redhat.com/browse/OSSM-1668[OSSM-1668]
-`jwksResolverCA` field is missing in `SMCP`.
-+
-If you upgrade from Service Mesh operator 2.1.3 to Service Mesh operator 2.2, then the `jwksResolverCA` field is not supported.
+`jwksResolverCA` field is missing in `SMCP`. If you upgrade from Service Mesh operator 2.1.3 to Service Mesh operator 2.2, then the `jwksResolverCA` field is not supported.
```

Comment:
Unless you are EXTREMELY familiar with the schema of the SMCP, I don't think this  description is providing enough information about the differences in the paths to the two `jwksResolverCA` fields.   I’ve tried to come up with a revision that 1) avoids too much confision around with OSSM versions have each field (since it’s pretty unclear in JIRA) and 2) provides the full paths for both versions of the field.
***

A new field `spec.security.jwksResolverCA` was added to the Version 2.1 SMCP but is missing in the 2.2.0 and 2.2.1 releases.  When upgrading from an Operator version where this field was present to an Operator version that is missing this field, the `.spec.security.jwksResolverCA`  field is not available in the SMCP.

Workaround:  To enable additional JWKS CA certificates when using an Operator version that is missing  `spec.security.jwksResolverCA`, the workaround is to use the `spec.techPreview jwksResolverExtraRootCA` field.

Reply:
@jwendell  and @skondkar, could you please take a look at my suggested revisions.

Reply:
This change should not be part of this PR.

Reply:
@jwendell I can revise this as I'm going to touch the file again. Do you think the suggested revision seems reasonable? It's a small piece of content to change, and I think the revision will help our customers.

Reply:
I'm not the best person to review this jwksResolverCA stuff. All I'm saying is that IMO this change should be done in a different PR as it does not relate with the OSSMDOC-601.

Reply:
@jwendell I broke this change out into OSSMDOC-650 and issued a PR. I asked Yuanlin to review it as he previously worked on the related dev ticket.